src/conf.c

```diff
@@ -66,9 +66,10 @@
 #define PASSWORD "([^@]*)"
 #define IP "((([0-9]{1,3})\\.){3}[0-9]{1,3})"
 #define IPMASK "(" IP "(/" DIGIT "+)?)"
+#define IPV6_SCOPE "((%[^ \\t\\/]{1,16})?)"
```

Comment:
`\t` does not need another backslash escape.

This is just a POSIX regular expression so it excludes a literal `\` and `t`. A scope like `%eth1` fails because of the `t`.

Reply:
good catch.

Reply:
Seems I did too much perl and my posix regex is a bit rusty.

Reply:
Fixed in 15c64e70f795512ec0bfc4074d91d724030c7691